package.json

```diff
@@ -19,10 +19,22 @@
     "cli-table": "^0.3.1",
     "ethers": "^3.0.22",
     "fs-extra": "^6.0.1",
-    "ganache-cli": "^6.1.4",
+    "ganache-cli": "6.1.4",
     "moment": "^2.22.2",
     "readline-sync": "^1.4.9",
-    "yargs": "^11.0.0"
+    "yargs": "^11.0.0",
+    "ajv": "^6.5.2",
+    "async": "^2.6.1",
+    "async-each-series": "^1.1.0",
+    "async.whilst": "^0.5.2",
+    "colors": "^1.3.0",
+    "ethjs-abi": "^0.2.1",
```

Comment:
Can we remove ethjs-abi

Reply:
I think we can't. It's used in contract.js and it's a part of the whole workflow